lib/external/io.dart

```diff
@@ -9,6 +9,8 @@ import 'package:Birb/utils/scope.dart';
 void registerIO(Runtime runtime) {
   registerGlobalFunction(runtime, 'exit', funcExit);
   registerGlobalFunction(runtime, 'free', funcFree);
+  registerGlobalFunction(runtime, 'getCwd', funcGetCwd);
+  registerGlobalFutureFunction(runtime, 'system', funcSystem);
```

Comment:
the name `system` doesn't give any information about what the function is intended to do